<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameDialog</class>
 <widget class="QDialog" name="RenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename Step Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Step to rename (location on disk):</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditStepLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStepChooser">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string> to:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditRenamePackageFrom">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEditRenamePackageTo"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Rename Package name from:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Rename Step name from:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string> to:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditRenameStepTo"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditRenameStepFrom">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidgetRename">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonRename">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditStepLocation</tabstop>
  <tabstop>pushButtonStepChooser</tabstop>
  <tabstop>lineEditRenameStepFrom</tabstop>
  <tabstop>lineEditRenameStepTo</tabstop>
  <tabstop>lineEditRenamePackageFrom</tabstop>
  <tabstop>lineEditRenamePackageTo</tabstop>
  <tabstop>treeWidgetRename</tabstop>
  <tabstop>pushButtonRename</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>RenameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
